src/cli-validator/utils/processConfiguration.js

```diff
@@ -367,9 +367,31 @@ const validateConfigOption = function(userOption, defaultOption) {
   return result;
 };
 
+const getSpectralRuleset = async function(defaultRuleset) {
+  const ruleSetFileName = '.spectral.json';
```

Comment:
I think we'd improve adoption if we supported yaml in addition or even instead of json.  Spectral strongly favors yaml for its configuration file -- all it's examples are shown in yaml.

Reply:
I'll add support for both a custom yaml and json ruleset file.